python/packages/autogen-ext/src/autogen_ext/task/_agent_name_termination.py

```diff
@@ -0,0 +1,38 @@
+from typing import Sequence, List
+
+from autogen_agentchat.base import TerminationCondition, TerminatedException
+from autogen_agentchat.messages import StopMessage, AgentMessage, ChatMessage
+
+
+class AgentNameTermination(TerminationCondition):
```

Comment:
Feel like it should be `SourceMatchTermination` to highlight the event that triggers the termination

Reply:
Yeah. That makes sense. i will update class name and file name on the next pull request. thank you

Reply:
Possible to update it in this PR?

Reply:
Sorry, i misstyped, i mean next commit 😀